Dockerfile

```diff
@@ -1,16 +1,16 @@
-FROM fedora:32
+FROM quay.io/centos/centos:stream
```

Comment:
This should be `stream8` - they're pointing to the same image right now, but I guess that might change once images for CentOS Stream 9 start coming out. https://www.centos.org/centos-stream/ specifically tells people to use the `stream8` tag.